sample.tf

```diff
@@ -0,0 +1,76 @@
+terraform {
```

Comment:
You're definitely going to want to include an example of how it integrates with the other providers